.semaphore/semaphore.yml

```diff
@@ -74,12 +74,7 @@ blocks:
             - checkout
             - make test
         run:
-          when:
-            event:
-              only: main
-            changeset:
-              comment:
-                matches: "LGTM"
+          when: "branch" = "master" OR "comment" = "LGTM"

```

Comment:
It doesn't work for a comment - https://docs.semaphoreci.com/reference/conditions-reference/